docs/source/notes/numpy_compatibility.rst

```diff
@@ -0,0 +1,388 @@
+.. _numpy_compatibility:
+
+.. testsetup:: *
+
+   import numpy as np
+   import torch
+
+NumPy Compatibility
+===================
+
+.. note::
+
+   **TL;DR:**
+
+   - Be explicit about obtaining and using NumPy arrays
```

Comment:
Having now read the document, I like being clear about converting between torch tensors and NumPy arrays. I think we should just tell people if they want to call NumPy operations on PyTorch tensors how they can do that (like creating a NumPy view of the data and running in a no grad context). We should let people know that PyTorch has a very "NumPy compatible" API and try to note the biggest divergences. We should talk about our Python Array API support (which is still in development) and let the user know if they can acquire the Python array API namespace from a torch tensor (not at this time).